src/host/exe/Host.EXE.vcxproj

```diff
@@ -84,8 +84,9 @@
     </ClCompile>
     <Link>
       <AllowIsolation>true</AllowIsolation>
+      <AdditionalDependencies>d3dcompiler.lib</AdditionalDependencies>

```

Comment:
There's probably a better place to put this, @miniksa knows where.

Reply:
(this one is still here too)